### PR TITLE
Fix Getting error type

### PR DESCRIPTION
**Pull request by tuxos**

### CODE REVIEW

File: facebook/__init__.py [hunk 1]
```diff
@@ -249,6 +249,7 @@ def request(
             raise GraphAPIError(response)
 
         headers = response.headers
+        result = {}
```
Comment (martey) [inline facebook/__init__.py]: Is this necessary? If we get down to `if result and isinstance(result, dict) and result.get("error"):`, shouldn't `result` always be defined?

File: facebook/__init__.py [hunk 2]
```diff
@@ -335,13 +334,14 @@ def __init__(self, result):
             # OAuth 2.0 Draft 00
             try:
                 self.message = result["error"]["message"]
+                if not self.type:
+                    self.type = result["error"].get('type', '')
```
Comment (martey) [inline facebook/__init__.py]: Earlier, `self.type` is defined as `result["error_code"]`. Is it possible that this change could overwrite that?

Comment (saloua-cliqz) [inline facebook/__init__.py]: This is executed only if `self.type` is either an empty string or None. And as `error_code` field is returning an int that should not be the case.

File: facebook/__init__.py (hunk 1 above)
Comment (saloua-cliqz) [inline facebook/__init__.py]: No it is not necessary.
That was added in order to have `result`  declaration/scope in the function body.

File: facebook/__init__.py (hunk 2 above)
Comment (martey) [inline facebook/__init__.py]: I think these two lines are the only ones that need to be included in the pull request. Can you create a new pull request with them?

**Pull request closed without merging**